website/content/docs/fork-workflow.md

```diff
@@ -26,18 +28,18 @@ When using the GitHub backend, you can use Netlify CMS to accept contributions f
 
 ## Using the Fork Workflow
 
-When a user logs into the CMS who doesn't have write access to your repo, the CMS will ask them if it's all right to create a fork of your repo (or use their existing fork, if they already have one). They are then presented with the normal CMS interface. The published content shown is from the original repo, so it stays up-to-date as changes are made.
+When a user logs into Netlify CMS who doesn't have write access to your repo, the CMS will ask for permission to create a fork of your repo (or use their existing fork, if they already have one). They are then presented with the normal CMS interface. The published content shown is from the original repo, so it stays up-to-date as changes are made.
 
-On the editorial workflow screen, the normal three columns will be replaced by two columns instead - "Draft" and "Ready to Review".
+On the editorial workflow screen, the normal three columns will be replaced by two columns instead — "Draft" and "Ready to Review".
 
-When they make changes to content in the CMS, the changes will be made to a branch on their fork. In the editorial workflow screen, they will only be able to see their pending changes. Once they're ready to submit their changes, they can move the card into the "Ready To Review" or "Ready to Publish" columns to create a pull request. When the pull request is merged, the CMS will delete the branch and remove the card the next time the user opens the editorial workflow screen. Fork workflow users will not be able to publish entries through the CMS.
+When they make changes to content in the CMS, the changes will be made to a branch on their fork. In the editorial workflow screen, they will see only their own pending changes. Once they're ready to submit their changes, they can move the card into the "Ready To Review" column to create a pull request. When the pull request is merged (by a repository maintainer via the GitHub UI), the CMS will delete the branch and remove the card the next time the user opens the editorial workflow screen. Fork workflow users will not be able to publish entries through the CMS.
```

Comment:
Not showing in the diff, but I removed the "Ready to Publish" column option, since the paragraph before says that's not there.